### PR TITLE
Documentation and some

### LINKED ISSUE #2039: Organization creation
- [x] store the organization data in db
- [x] store the review-email address
- [x] minimal CRUD UI 
  - [x] list
  - [x] view
  - [x] create
  - [x] edit (emails+)
- store members as they log-in through invite links?
- name and description for organization used in the organization dropdowns?
- [x] localize organization name
- [x] localize review email name
- [x] make organizations unoptional in API user return values
- [x] short name i.e. abbreviation for organization names?
- [x] browser test the organization UI
- [ ] check documentation and installation instructions

### PR DESCRIPTION
**Pull request by Macroz**
Closes #2039 

- improve the docs regarding organizations
- improve docs of bots (how to create the users)
- installing and upgrading clarifications
- fix typo uusi lisessi
- cleaner error message in UI to admin for missing orgs

org_not_found

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue
- [x] consider adding screenshots for ease of review

## API
- [x] API is backwards compatible or completely new

## Documentation
- [x] update changelog if necessary

### CODE REVIEW

File: docs/installing-upgrading.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ This is a short guide for running REMS as an uberjar. We also provide a [Dockerf
 
 # Installing REMS
 
-1. Set up a PostgreSQL database.
+1. Set up a PostgreSQL database and create the data model with e.g. `migrate` command.
```
Comment (opqdonut) [inline docs/installing-upgrading.md:5]: perhaps running `migrate` should be the second-to-last step? you need the rems.jar and the configuration file.

File: src/clj/rems/api.clj [hunk 2]
```diff
@@ -72,6 +74,12 @@
   (-> (service-unavailable "please try again")
       (plain-text)))
 
+(defn data-exception-handler
+  [exception _ex-data _request]
+  (log/error "data exception" (pr-str (.-data exception)))
+  (-> (service-unavailable (json/generate-string (.-data exception)))
```
Comment (opqdonut) [inline src/clj/rems/api.clj:80]: nice

Comment (opqdonut) [reply src/clj/rems/api.clj:80]: how about one api-level test that exercises this?

**Review by opqdonut — CHANGES_REQUESTED** [review]

File: docs/installing-upgrading.md (hunk 1 above)
Comment (opqdonut) [reply docs/installing-upgrading.md:5]: or you could consider combining these instructions with the Upgrading REMS section below

File: src/clj/rems/api.clj (hunk 2 above)
Comment (Macroz) [reply src/clj/rems/api.clj:80]: yeah I can make one, also perhaps later we'll have other DataExceptions

File: docs/installing-upgrading.md (hunk 1 above)
Comment (Macroz) [reply docs/installing-upgrading.md:5]: Maybe this shouldn't be updated now. Looks like we could have a slightly simplified docker compose setup soon or at least we could combine the instructions here?

Comment (opqdonut) [reply docs/installing-upgrading.md:5]: yeah let's not update this now

**Review by opqdonut — APPROVED** [review]
see my comment about not updating installing-and-upgrading.md right now

**Pull request merged**